configs/JSC-JUWELS-Booster-rome-A100-Infiniband/auxiliary_software_files/compilers.yaml

```
@@ -0,0 +1,34 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+compilers:
+- compiler:
+      spec: nvhpc@=23.7
+      paths:
+        cc: /p/software/juwelsbooster/stages/2024/software/NVHPC/23.7-CUDA-12/Linux_aarch64/23.7/compilers/bin/nvc
+        cxx: /p/software/juwelsbooster/stages/2024/software/NVHPC/23.7-CUDA-12/Linux_aarch64/23.7/compilers/bin/nvc++
+        f77: /p/software/juwelsbooster/stages/2024/software/NVHPC/23.7-CUDA-12/Linux_aarch64/23.7/compilers/bin/nvfortran
+        fc: /p/software/juwelsbooster/stages/2024/software/NVHPC/23.7-CUDA-12/Linux_aarch64/23.7/compilers/bin/nvfortran
+      operating_system: rocky9
+      target: aarch64
```

```suggestion
      target: x86_64
```

JUWELS Bosoter is using AMD Zen2 which are x86_64.